mne/viz/backends/_pysurfer_mayavi.py

```diff
@@ -128,7 +128,7 @@ def mesh(self, x, y, z, triangles, color, opacity=1.0, shading=False,
 
     def contour(self, surface, scalars, contours, line_width=1.0, opacity=1.0,
                 vmin=None, vmax=None, colormap=None,
-                normalized_colormap=False):
+                normalized_colormap=False, tube=False, color=None):
```

Comment:
`tube` seems to parametrize the size of the tube. Basically I see it can be a bool or an int. Is it different from line_width? also seeing a new param without any docstring update triggers a "why?".